packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDetailsSteps.tsx

```diff
@@ -231,6 +256,61 @@ export const TransactionsTableDetailsSteps = ({
         }
       />
 
+      {/* show mid transaction step for teleport tx */}
```

Comment:
In transaction history steps, we add 2 additional steps for teleporter transactions.
1. For showing the mid layer 2 transaction
2. For showing the wait time for L2 to L3 transaction